Detectors/ITSMFT/ITS/postprocessing/workflow/standalone-postprocessing-workflow.cxx

```diff
@@ -63,12 +63,13 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   o2::globaltracking::InputHelper::addInputSpecsPVertex(configcontext, specs, useMC);
   o2::globaltracking::InputHelper::addInputSpecsSVertex(configcontext, specs);
 
+  // additional sources
+  specs.emplace_back(o2::steer::getMCKinematicReaderSpec());
```

Comment:
is it a hard requirement that you need to include devices likes this? Maybe it could be sufficient to do this at runtime using DPL pipes.